src/components/Icon/index.tsx

```diff
@@ -33,7 +33,7 @@ const ICONS = {
     <path d="M12 16.5a1.58 1.58 0 0 1-1.116-.458l-5.921-5.87a1.556 1.556 0 0 1 0-2.213 1.588 1.588 0 0 1 2.232 0L12 12.722l4.804-4.763a1.59 1.59 0 0 1 2.234 0 1.557 1.557 0 0 1 0 2.213l-5.921 5.87A1.584 1.584 0 0 1 12 16.5" />
   ),
   'arrow-left': () => (
-    <path d="M5.29289 9.2823C4.90237 8.89178 4.90237 8.25861 5.29289 7.86809L9.29289 3.86809C9.68342 3.47756 10.3166 3.47756 10.7071 3.86809C11.0976 4.25861 11.0976 4.89178 10.7071 5.2823L7.41421 8.5752L10.7071 11.8681C11.0976 12.2586 11.0976 12.8918 10.7071 13.2823C10.3166 13.6728 9.68342 13.6728 9.29289 13.2823L5.29289 9.2823Z" />
+    <path d="M5.29289 8.70735C4.90237 8.31683 4.90237 7.68366 5.29289 7.29314L9.29289 3.29314C9.68342 2.90261 10.3166 2.90261 10.7071 3.29314C11.0976 3.68366 11.0976 4.31683 10.7071 4.70735L7.41421 8.00024L10.7071 11.2931C11.0976 11.6837 11.0976 12.3168 10.7071 12.7074C10.3166 13.0979 9.68342 13.0979 9.29289 12.7074L5.29289 8.70735Z" />
```

Comment:
Isn't this only precision related ?

Reply:
I took this one because we are updating all ours icons so need to be aligned with design ? Sometimes there are 1 point diff